src/infrastructure/figma/figma-client/types.ts

```diff
@@ -108,3 +108,13 @@ export type CreateWebhookResponse = {
 	readonly description: string | null;
 	readonly protocol_version: string;
 };
+
+export type Project = {
+	id: string;
```

Comment:
**suggestion:** Consider adding the `readonly` modifier for consistency with other models.